opnsense/bridge/Makefile

```diff
@@ -0,0 +1,33 @@
+PORTNAME=	bridge
+PORTVERSION=	1.0
+CATEGORIES=	sysutils
+DISTFILES=
+
+MAINTAINER=	stephan.de.wit@deciso.com
+COMMENT=	Bridge two network interfaces in netmap mode
+
+LICENSE=	BSD
+
+ALL_TARGET=	bridge
+MANDIRS=	${PREFIX}/man/man8
+
+.include <bsd.port.options.mk>
+
+pre-extract:
+.if !exists(${SRC_BASE}/tools/tools/netmap/Makefile)
```

Comment:
the only issue this has is that PORTVERSION is not really indicative of what version is inside

Reply:
it's bogus, yes. I'll omit it while renaming the port.